Project.toml

```diff
@@ -3,11 +3,9 @@ uuid = "7f8f8fb0-2700-5f03-b4bd-41f8cfc144b6"
 version = "0.3.0"
 
 [deps]
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
```

Comment:
Where is Markdown used?

Reply:
It is used in the docstrings for the LaTeX rendering. The code doesn't compile otherwise.

Reply:
instead of `@doc doc"""..."""` I believe `@doc raw"""..."""` does the same thing without the need of Markdown.

Ref: https://docs.julialang.org/en/v1/manual/documentation/#and-\\-characters-1